test/unit/spec/v1/VerifyPIV_spec.js

```diff
@@ -135,6 +137,49 @@ Expect.describe('PIV', function() {
           });
         });
     });
+    itp('makes post call with correct data when isCustomDomain is false and customDomain is set', function() {
+      const config = {
+        certAuthUrl: 'https://foo.com',
+        isCustomDomain: false,
+        customDomain: 'bar.com',
+      };
+
+      return setup(null, config)
+        .then(function() {
+          return Expect.waitForSpyCall(SharedUtil.redirect);
+        })
+        .then(function() {
+          expect(Util.numAjaxRequests()).toBe(2);
+          const argsForPost = Util.getAjaxRequest(1);
+
+          expect(JSON.parse(argsForPost.params)).toEqual({
+            fromURI: '%2Fapp%2FUserHome',
+            isCustomDomain: false,
+            customDomain: 'bar.com',
+          });
+        });
+    });
+    itp('makes post call with correct data when isCustomDomain is false and customDomain is undefined', function() {
+      const config = {
+        certAuthUrl: 'https://foo.com',
+        isCustomDomain: false,
+        customDomain: undefined,
```

Comment:
nit: you don't need `customDomain: undefined,`

Reply:
Is it because `customDomain` would be `undefined` anyway if I don't declare it?